NEWS.md

```diff
@@ -27,6 +27,8 @@
   more informative warnings when supplied with set aesthetics
   (i.e., `slope`, `intercept`, `yintercept`, and/or `xintercept`)
   and mapped aesthetics (i.e., `data` and/or `mapping`).
+  
+* Added link to `labs()` and `lims()` to `scale_(x|y)_discrete` (#3301, @Maschette)
```

Comment:
This is a documentation issue so doesn't need a news bullet (i.e. you can delete this)